ecc/bls12-377/ecdsa/doc.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 ConsenSys Software Inc.
+// Copyright 2020 Consensys Software Inc.
```

Comment:
Should this change to 2023 as well or is it supposed to be the original publishing date?

Reply:
normally, it should reflect the initial creation date.